.github/workflows/build.yml

```diff
@@ -406,7 +407,7 @@ jobs:
         if: matrix.use_llvm != true && !endsWith(matrix.build, 'v8') && !endsWith(matrix.build, 'wamr') && !endsWith(matrix.build, 'wasmi')
         shell: bash
         run: |
-          make build-wasmer
+          ENABLE_LLVM=0 make build-wasmer
```

Comment:
I'm not sure why, but at this step, wasmer is being built with the llvm feature enabled. You can see it on [main](https://github.com/wasmerio/wasmer/actions/runs/11913682684/job/33199952757) for example.
Since LLVM is not available, it causes the musl build to fail. This is sort of a hot fix to force disable the LLVM feature when `use_llvm` is `false`.